sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -368,7 +369,74 @@ size_t gnrc_sixlowpan_iphc_decode(gnrc_pktsnip_t *ipv6, gnrc_pktsnip_t *pkt, siz
 
     }
 
-    /* TODO: add next header decoding */
+    /* next header decoding */
```

Comment:
I'd prefer to have this whole chunk of code in a seperate module `gnrc_sixlowpan_nhc`, so one can turn it of.